docs/development/application-logic.md

```diff
@@ -287,6 +290,48 @@ benefits-->>analytics: returned enrollment
     deactivate benefits
 ```
 
+### Switchio
+
```

Comment:
Related to the other comment, can we link here to the entrypoint for Switchio (views I guess?)